### ISSUE TITLE
Not getting similar results as reported in the paper

### ISSUE THREAD

**Issue by ry85** [mhw32/multimodal-vae-public#3]
Code is not working for celeba. I am not able to obtain the results as reported in the paper. The image reconstruction and random sample generation both are of worse quality. Looks like model is not getting trained properly. Can you please provide correct running implementation.

**Comment by gabinsane** [comment]
I can confirm that the model does not converge for the MNIST datasets either. This is the training curve for basic MNIST, trained with the default parameters. 
![MNIST_loss](https://user-images.githubusercontent.com/38910946/64543325-4f591680-d325-11e9-9155-0855399c017e.png)

**Comment by liuyangdh** [comment]
> Code is not working for celeba. I am not able to obtain the results as reported in the paper. The image reconstruction and random sample generation both are of worse quality. Looks like model is not getting trained properly. Can you please provide correct running implementation.

Me too, did you try the code in the [bugfix](https://github.com/mhw32/multimodal-vae-public/tree/bugfix) branch?

**Comment by ry85** [comment]
> 
> 
> > Code is not working for celeba. I am not able to obtain the results as reported in the paper. The image reconstruction and random sample generation both are of worse quality. Looks like model is not getting trained properly. Can you please provide correct running implementation.
> 
> Me too, did you try the code in the [bugfix](https://github.com/mhw32/multimodal-vae-public/tree/bugfix) branch?

Yes, I have tried that. Its even worst, giving NaN in the first epoch itself.

**Comment by poonehmousavi** [comment]
> > > Code is not working for celeba. I am not able to obtain the results as reported in the paper. The image reconstruction and random sample generation both are of worse quality. Looks like model is not getting trained properly. Can you please provide correct running implementation.
> > 
> > 
> > Me too, did you try the code in the [bugfix](https://github.com/mhw32/multimodal-vae-public/tree/bugfix) branch?
> 
> Yes, I have tried that. Its even worst, giving NaN in the first epoch itself.

did you manage to figure out what is the problem? I also got NAN when running code in fix branch
